src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -89,6 +90,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case FilterTagCommand.COMMAND_WORD:
             return new FilterTagCommandParser().parse(arguments);
 
+        case TagCommand.COMMAND_WORD:
```

Comment:
Perhaps can add `parseCommand_tag` test in `AddressBookParserTest` to cover it by test